core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1063,6 +1063,14 @@ public enum Property {
       "1.3.5"),
   TABLE_ARBITRARY_PROP_PREFIX("table.custom.", null, PropertyType.PREFIX,
       "Prefix to be used for user defined arbitrary properties.", "1.7.0"),
+  TABLE_MINC_INPUT_DROP_CACHE_EXCLUSIONS("table.compaction.minor.output.drop.cache.exclusions", "",
```

Comment:
Thoughts on dropping the "minor" from the property/description? That had me thinking we were talking about flushes initially.